HW03_Xingruo_Summer_Zhang.Rmd

```diff
@@ -0,0 +1,69 @@
+---
+title: "HW03_Xingruo_Summer_Zhang"
+author: "Summer Zhang"
+date: "7/20/2020"
+output: md_document
+---
+
+```{r}
+# Load Packages
+library(ggplot2)
+library(readr)
+library(magrittr)
+library(tidyr)
+library(dplyr)
+
+# Read Data
+us <- read_csv("us.csv")
+states <- read_csv("us-states.csv")
+```
+
+```{r}
+head(states)
+```
+
+## Graph 1: How Caes in the Entire U.S. Grew
+```{r}
+# Graph the cases
```

Comment:
This is a nice graph! It might be helpful to change the scientific notation on y-axis to the original number.